lib/Walli.ts

```diff
@@ -54,7 +67,7 @@ class Walli {
     }
   }
 
-  private logCouldNotConnect = (service: string, error: any) => {
+  private logCouldNotConnect = (service: string, error?: any) => {
```

Comment:
`error?` should never be `undefined` or `null` and therefore it should stay `error`